docs/source/How_to_install_liminal_in_airflow_on_kubernetes.md

```diff
@@ -0,0 +1,160 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required bgit y applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Install Liminal in Airflow
+* [Workflow](#workflow)
+* [Prerequisites](#prerequisites)
+   * [Supported Distributions](#supported-distributions)
+* [Get Started](#Get-Started)
+   * [Scripted Installation](#scripted-installation)
+   * [Manual Installation](#manual-installation)
+* [References and other resources](#references-and-other-resources)
+
+## Workflow
+Liminal deploys into the Airflow components and the Yamls home folder is mounted by EFS.
+The process of liminal is emphasized on the following diagram:
+
+![](assets/liminal-diagram.png)
+
+## Prerequisites
+Before you begin, ensure you have met the following requirements:
+
+* You have a Kubernetes cluster running with [Helm][homebrew-helm] (and Tiller if using Helm v2.x) installed
+* You have the kubectl command line [(kubectl CLI)][homebrew-kubectl] installed
+* You have the current [context][cluster-access-kubeconfig] in kubernetes' kubeconfig file
+* You have [Airflow on Kubernetes with AWS EFS][airflowInstallation]
```

Comment:
this post is actually not good in my opinion.
I believe the recommended way to deploy efs today is to use csi;
in this post they for some reason set up an NFS server pod, not sure what for...

Essentially I think we can write our own post in a matter of 2-3 hours which will be better.

Reply:
I guess it will be the best to do so